cluster/gce/gci/configure-helper.sh

```diff
@@ -1606,11 +1606,10 @@ EOF
 #
 # $1 is the file to create.
 # $2: the log owner uid to set for the log file.
-# $3: the log owner gid to set for the log file.
 function prepare-log-file {
   touch "$1"
-  chmod 644 "$1"
```

Comment:
Will this break automations that read these logs as non-root?

nit: also, as a developer, now I need to sudo to read these logs when debugging in the master.